jobs/JGLOBAL_WAVE_POST_SBS

```diff
@@ -9,9 +9,15 @@ export errchk=${errchk:-err_chk}
 export MP_PULSE=0
 
 # Set COM Paths and GETGES environment
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_WAVE_PREP COM_WAVE_HISTORY COM_WAVE_GRID
-
-mkdir -p "${COM_WAVE_GRID}"
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+    COMIN_WAVE_PREP:COM_WAVE_PREP_TMPL \
+    COMOUT_WAVE_PREP:COM_WAVE_PREP_TMPL \
```

Comment:
```suggestion
```

Reply:
What is going out in `COMOUT_WAVE_PREP` from this job?

Reply:
@WalterKolczynski-NOAA tells me that the interpolation grids are written out to `COMOUT_WAVE_PREP` if they are not available.  Perhaps those interpolation grids should be available or generated in `JGLOBAL_WAVE_PREP`.  That is outside the scope of this PR.
Check w/ @JessicaMeixner-NOAA if that is correct.

Reply:
Yeah, but I wouldn't think interpolation girds should be written to waveprep, that just seems weird.

Reply:
I agree with your assessment, and we can revisit this at a later point @JessicaMeixner-NOAA.

Reply:
Made a note in https://github.com/NOAA-EMC/global-workflow/issues/297